Form/Handler.php

```diff
@@ -132,6 +149,10 @@ public function handle(FormInterface $form, $attributes = [])
             return false;
         }
 
+        if (!$this->checkSum($form)) {
```

Comment:
the checksum check should be done in the `buildByRequest` and not in the FormHandler